<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Port/Baudrate Selection</string>
  </property>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>20</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="portsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select Port</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="baudratesList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>500</width>
     <height>50</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <item>
    <property name="text">
     <string>Select Baudrate</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
